O/OpenFHE/common.jl

```diff
@@ -0,0 +1,84 @@
+using BinaryBuilder, Pkg
+
+function prepare_openfhe_build(name::String, git_hash::String)
+    # Collection of sources required to complete build
+    sources = [
+        GitSource("https://github.com/openfheorg/openfhe-development.git",
+                  git_hash),
+        DirectorySource("./bundled")
+    ]
```

Comment:
Instead of creating a symlink to the `bundled` directory, could you just modify the directory source here? That is, use something like
```suggestion
    # Collection of sources required to complete build
    # Note: The directory source is made to work for both OpenFHE and OpenFHE_128 builds
    sources = [
        GitSource("https://github.com/openfheorg/openfhe-development.git",
                  git_hash),
        DirectorySource("../OpenFHE/bundled")
    ]
```
If I'm not mistaken, this should work for either library build?

Reply:
That is really a cool idea! It works! Thank you.